user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures/Resource_migration_to_elastic.md

```diff
@@ -59,15 +61,15 @@ To migrate the resources, you will need to use the SLNetClientTest tool.
 
 If the migration should fail for any reason, the migration status object in the SLNetClientTest tool window will get a red background color. The ``SLMigrationManager.txt`` and ``SLResourceManager.txt`` log files will contain more information. Resource Manager will not switch the configuration, so XML storage will still be used after a failed migration.
 
-If a ``MigrationStatus`` is stuck in the ``InProgress`` state, you will need to cancel the migration to make all Resource Manager instances start or to trigger the migration again. You can do so with the *Cancel Migration* button in the SLNetClientTest tool window.
+If a ``MigrationStatus`` is stuck in the ``InProgress`` state, you will need to cancel the migration to make all Resource Manager instances start or to trigger the migration again. You can do so with the *Cancel Migration* button in the *Resources XML to Elastic* section of the SLNetClientTest tool window.
 
 ### Behavior in new installations
 
 When a new DataMiner Agent is installed, the used storage type will depend on when Resource Manager starts up for the first time.
 
 - If DataMiner is installed with the 10.2.0 installer and Resource Manager is used, XML storage will be used. Elasticsearch is not yet supported as a storage type for resources and resource pools in DataMiner 10.2.0. After you have upgraded this DataMiner Agent to DataMiner 10.3.0 or later, it will continue to use XML storage until you trigger the migration.
 
-- If DataMiner is installed with the 10.2.0. installer but Resource Manager never starts up while this version is used (for example because Elasticsearch is not installed, which is a requirement for Resource Manager to start), and the DataMiner Agent is then upgraded to 10.3.1, Resource Manager will use Elasticsearch storage when it is initialized.
+- If DataMiner is installed with the 10.2.0. installer but Resource Manager never starts up while this version is used (for example because Elasticsearch is not installed, which is a requirement for Resource Manager to start as Elasticsearch is used to store bookings), and the DataMiner Agent is then upgraded to 10.3.1, Resource Manager will use Elasticsearch storage when it is initialized.
```

Comment:
```suggestion
- If DataMiner is installed with the 10.2.0 installer but Resource Manager never starts up while this version is used (for example because Elasticsearch is not installed, which is a requirement for Resource Manager to start as Elasticsearch is used to store bookings), and the DataMiner Agent is then upgraded to 10.3.1, Resource Manager will use Elasticsearch storage when it is initialized.
```